server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/TikaMailboxModule.java

```diff
@@ -48,13 +51,29 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(TikaMailboxModule.class);
 
     private static final String TIKA_CONFIGURATION_NAME = "tika";
+    private static final String TEXT_EXTRACTOR_NAME = "text_extractor";
```

Comment:
I don't agree, migration would be a hell

Reply:
What migration did you expect?

We need this feature because Tika is failing to parse some content.
For me it's related to Tika & should be in it's configuration file.

Reply:
Deciding what content type we don't want to extract text from is not related to tika.
We use it only in tika right now but as soon as you have a second implementation, you need that blacklisting for both.
if we start by putting the configuration in tika config, we'll have to handle the move to another file the day we decide to extract it.
Given having a config file is pretty cheap, I don't see why we would put it in tika configuration (even if the file is actually handled by tika in the first place)

Reply:
I agree with your remark, but it seems a bit over-engineered for me.

Is there a plan for having another tool for text extracting, I think no.

Reply:
I already considered alternatives to tika (because it was slow in the past).
For example, on gnome desktop, there's a tool called `tracker-extract` with the exact same purpose.

Reply:
If there is a new TextExtractor impl, we could change `tika.properties` to an appropriate name like `textExtractor.properties` => for defining what kind of text extractor you want to use and specific configurations for it. Similar to choosing blobstore configuration. We need only one file `objectstore.properites`.

BTW, the purpose of this PR is simple. Could we consider this PR is be okay with current status? And then make another diccussion for that topic and then PRs.... ?

Reply:
> I don't agree, migration would be a hell

Which migration?

Honestly, we would add the same property in `otherTextExtractor.properties` and hop job done

IMO config file multiplication is also some hell we have to pay attention to.

Reply:
IMO keeping the contentType blacklist in Tika is good idea. 
First, We create `text-extractor.propeties` with only one configuration in this is quite confused. 
Second, we use each config file for each Text Extractor still keep the generic. For the case in the future we may be add more Text Extractors with more contentType (image, ...) are supported

Reply:
I didn't change my opinion, but well, I won't fight for that level of detail, it's not worth it.